build_tools/cmake/iree_hal_cts_test_suite.cmake

```diff
@@ -89,6 +89,15 @@ function(iree_hal_cts_test_suite)
       list(APPEND _TRANSLATE_FLAGS "--iree-llvm-target-triple=${_TARGET_TRIPLE}")
     endif()
 
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv" AND
+      RISCV_CPU STREQUAL "rv64" AND
+      NOT  _TRANSLATE_FLAGS MATCHES "iree-llvm-target-triple")
```

Comment:
There is no need to check `_TRANSLATE_FLAGS` here, it is set directly just above this (not coming in from function args).

The comment below is also verbose and can be removed entirely IMO.

Reply:
Thanks. I will clean up in a followup PR.